js/components/closed-caption/fontSizeTab.js

```diff
@@ -40,24 +42,38 @@ var FontSizeTab = React.createClass({
       this.props.localizableStrings
     );
     var fontItems = [];
-    for(var i = 0; i < this.state.fontSizes.length; i++) {
+    for (var i = 0; i < this.state.fontSizes.length; i++) {
       //accent color
+      var isSelected = this.props.closedCaptionOptions.fontSize === this.state.fontSizes[i];
       var selectedFontSizeStyle = {};
-      if (this.props.closedCaptionOptions.enabled && this.props.skinConfig.general.accentColor && this.props.closedCaptionOptions.fontSize == this.state.fontSizes[i]) {
+      if (this.props.closedCaptionOptions.enabled && this.props.skinConfig.general.accentColor && isSelected) {
         selectedFontSizeStyle = {color: this.props.skinConfig.general.accentColor};
       }
+      var itemLabel = Utils.getLocalizedString(
+        this.props.language,
+        CONSTANTS.SKIN_TEXT[this.state.fontSizes[i].toUpperCase().replace(" ", "_")],
```

Comment:
Right now there are fontSizes: ["Small", "Medium", "Large", "Extra Large"] defined in the initial state and the same values in the SKIN_TEXT constants. Can we combine these somehow?

Reply:
I can look into this if you want, but this is just the old code which I moved for readability. I'm not sure why it was implemented this way.